Gopkg.lock

```diff
@@ -1694,6 +1695,7 @@
     "github.com/docker/go-metrics",
     "github.com/ghodss/yaml",
     "github.com/gobuffalo/packr/v2",
+    "github.com/gobuffalo/packr/v2/file/resolver",
```

Comment:
The build is going to yell at you for this line. If you run `make clean-packr`, then run `dep ensure` again, the build will be happy. 

If you run `make verify` locally, you will find errors like this before the CI does. The reason why is that we don't check in packr generated files that don't need to be there, but dep picks them up if you have them present. To avoid this in the future run `make clean-packr` before running dep.